File: day2/doit.py
points = {
    'X': {
        'A': 3,
        'B': 1,
        'C': 2,
     },
    'Y': {
        'A': 1 + 3,
        'B': 2 + 3,
        'C': 3 + 3,
     },
    'Z': {
        'A': 2 + 6,
        'B': 3 + 6,
        'C': 1 + 6,
     },
}

f = open('input.txt', 'r')

score = 0

for line in f:
    parts = line.strip().split(' ')
    score += points[parts[1]][parts[0]]

f.close()

print(score)

File: day2/doit.js
var fs = require('fs');

const points = { X: { A: 3, B: 1, C: 2 }, Y: { A: 4, B: 5, C: 6 }, Z: { A: 8, B: 9, C: 7 } };

const score = fs.readFileSync('input.txt').toString().split('\n')
                .reduce((acc, line) => acc + points[line.split(' ')[1]][line.split(' ')[0]], 0);

console.log(score);